x/eibc/types/tx.pb.go

```diff
@@ -129,6 +133,162 @@ func (m *MsgFulfillOrderResponse) XXX_DiscardUnknown() {
 
 var xxx_messageInfo_MsgFulfillOrderResponse proto.InternalMessageInfo
 
+// MsgFulfillOrderAuthorized defines the FulfillOrderAuthorized request type.
+type MsgFulfillOrderAuthorized struct {
```

Comment:
were missing here denom and validation by denom.

Reply:
The denom is in the price, and is checked against the authorized denoms [**here**](https://github.com/dymensionxyz/dymension/blob/b7e114f943404f30997f5a6c285b38923b70dd93/x/eibc/types/fulfill_order_authorization.go#L71)